.circleci/gradle.properties

```diff
@@ -1,5 +1,5 @@
-org.gradle.jvmargs=-Xmx4g -Dkotlin.compiler.execution.strategy="in-process"
+org.gradle.jvmargs=-Xmx2560m -Dkotlin.compiler.execution.strategy="in-process"
```

Comment:
We need to make sure that parallel builds don't overstep memory so dropping total limit to half the container memory with 2 max workers.